03 Navigation/src/components/app.tsx

```diff
@@ -1,10 +1,5 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
-import { Router, Route, IndexRoute, Link, IndexLink, browserHistory, hashHistory  } from 'react-router'
-
-import Header from './common/header'
-import aboutPage from './about/aboutPage';
-import membersPage from './members/membersPage';
+import Header from './common/header';
 
 
 interface Props {
```

Comment:
Do I must extends from React.Props in all Props interfaces or only in this file?